src/gql.rs

```diff
@@ -1,19 +1,15 @@
-pub fn fetch_gql() -> String {
-    format!("
-{{
-    \"query\": \"
-    {{
-        bestPrices(departure: \\\"2020-07-21\\\", 
-          origin: \\\"POA\\\", 
-          destination: \\\"GRU\\\") {{
-          bestPrices {{
-            date
-            available
-            price {{amount}}
-          }}
-        }}
-      }}
-    \"
-}}
-    ")
-}
\ No newline at end of file
+use serde_json::{json, Value};
+
+pub fn fetch_gql() -> Value {
+    json!({
+        "query": "{
+             bestPrices(departure: \"2020-07-21\", origin: \"POA\", destination: \"GRU\") {
+                bestPrices {
+                    date
+                    available
+                    price {amount}
```

Comment:
This is the `amount` part I've told that it is probably wrong.

Reply:
The problem was I was using `format!` and not `json!`. This sucks kkkkk

Reply:
Yeah hhahahahha